templates/api/knowledge-panels/health/ingredients/nova.tt.json

```diff
@@ -44,10 +46,10 @@
                 "html": `
 <p>[% lang("nova_classification_description") %]</p>
 <ol>
-<li>[% lang('attribute_nova_1_description_short') %]</li>
-<li>[% lang('attribute_nova_2_description_short') %]</li>
-<li>[% lang('attribute_nova_3_description_short') %]</li>
-<li>[% lang('attribute_nova_4_description_short') %]</li>
+<li>[% IF nova_group == 1 %]<b>[% END %][% lang('attribute_nova_1_description_short') %][% IF nova_group == 1 %]</b>[% END %]</li>
```

Comment:
We can use a style, it will be cleaner and will halve the number of IF conditions.